src/domains/ecommerce-app/api/ecommerce-checkout/v1/_openapi.json.tpl

```diff
@@ -1558,6 +1436,135 @@
             "maxLength": 140
           }
         }
+      },
+      "PaymentOption": {
+        "type": "object",
+        "properties": {
+          "touchpoint": {
+            "type": "string"
+          },
+          "bin": {
+            "type": "string"
+          },
+          "idPspList": {
+            "type": "array",
+            "items": {
+              "type": "string"
+            }
+          },
+          "paymentAmount": {
+            "type": "integer",
+            "format": "int64"
+          },
+          "primaryCreditorInstitution": {
+            "type": "string"
+          },
+          "transferList": {
+            "type": "array",
+            "items": {
+              "$ref": "#/components/schemas/TransferListItem"
+            }
+          }
+        },
+        "required": [
+          "touchpoint",
+          "paymentAmount",
+          "primaryCreditorInstitution",
+          "transferList"
+        ]
+      },
+      "BundleOption": {
```

Comment:
it confuses me 😁 the dto models is `PaymentOption` with a field `paymentOption` of `Transfert` type?

Reply:
cc @ciuffagianluca @Fernando-Granato @MarcoAGreco

Reply:
Do we want to carry out a refactoring of names? Or do we postpone it to the next sprint ? @ciuffagianluca @infantesimone @MarcoAGreco